website/src/components/toc.page.tsx

```diff
@@ -1,34 +1,109 @@
+import { useState } from "react"
 import Link from "src/components/link"
-import { listItem, numberedOrderedList, orderedList } from "./toc.css"
+import { CollectionSection } from "src/graphql/dailp"
+import {
+  Chapter,
+  useChapters,
+} from "src/pages/edited-collections/edited-collection-context"
+import { useRouteParams } from "src/renderer/PageShell"
+import * as css from "./toc.css"
 
 type TOCData = {
   title: string
   path: string
   children?: TOCData[]
 }
 
-export const TOC = (props: {
-  introChapters?: TOCData[]
-  bodyChapters: TOCData[]
-}) => (
-  <>
-    <ol className={numberedOrderedList}>
-      {props.introChapters?.map((items, index) => (
-        <li>
-          <Link href={items.path}>{items.title}</Link>
-        </li>
-      ))}
-    </ol>
-    <ol className={orderedList}>
-      {props.bodyChapters.map((items, index) => (
-        <li className={listItem}>
-          <Link href={items.path}>{items.title}</Link>
-          {items.children ? <TOC bodyChapters={items.children}></TOC> : null}
-        </li>
-      ))}
-    </ol>
-  </>
-)
+interface Chapters {
```

Comment:
Once we merge the changes with the additional type of Credit chapter, there might have to be an additional type here

Reply:
Which I guess also brings up a different question I have: Are the different types of chapters mostly for organization within the spreadsheet and database? Since all chapters have the option to display a document, for instance, and the same fields, how relevant is distinguishing between them for display purposes? Not sure if that question makes sense but just some thoughts looking at the code brought up

Reply:
I think it's mostly for organization, so mainly a really semantic distinction, though they have the same structure!